picard.spec

```diff
@@ -163,6 +164,9 @@ else:
                 ],
                 'CFBundleTypeRole': 'Editor',
             }],
+            'CFBundleURLTypes': [{
+                'CFBundleURLSchemes': [PICARD_CUSTOM_PROTOCOL],
```

Comment:
https://developer.apple.com/documentation/bundleresources/information-property-list/cfbundleurltypes indicates CFBundleURLName and CFBundleURLIconFile as properties along CFBundleURLSchemes, it might be worth considering adding those too.

Reply:
For the installer version this is what Edge displays when opening such a link:

![grafik](https://github.com/user-attachments/assets/0a1c6522-713a-415a-ac29-2b502bdb8cd9)

No icon in this case, and the name shown is taken from the description inside the `picard.exe` file (which we set at https://github.com/metabrainz/picard/blob/master/win-version-info.txt.in#L39).

I have yet to test the store version (for which the above configuration is).

Reply:
The `CFBundleURLName` is actually called the "URL identifier". It is used to distinguish the case when multiple apps registered the same protocol and https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app#Register-your-URL-scheme suggest to set this to the reverse DNS name.

Not sure if this is needed when the protocol is already reverse DNS. But we could set it. I started testing on macOS and so far this is not yet properly working, so there will likely be some more changes.

Reply:
I did set the data. The logo though is unclear where or even if it is used at all. Same on Windows, btw.